fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/Repository.java

```diff
@@ -515,6 +515,27 @@ public void createRelationship(PID subject, Property property, Resource object)
 			throw ClientFaultResolver.resolve(e);
 		}
 	}
+	
+	/**
```

Comment:
Apparently I failed to add a description to createRelationship, oops!  Could you add a short description to createProperty and createRelationship?

Also, since the two methods are identical past the first line, you may want to abstract out a private method for them both.